validate_request.py

```diff
@@ -12,7 +12,6 @@
 from config import GS_BASE_URL, WEST_BBOX, EAST_BBOX, SEAICE_BBOX
```

Comment:
```suggestion
from config import WEST_BBOX, EAST_BBOX, SEAICE_BBOX
```
I think GS_BASE_URL is also no longer used in this module.